### PR TITLE
SO-98 Not Found Page

### PR DESCRIPTION
**Pull request by DGConn**
Not Found page

### CODE REVIEW

**Comment by hashemJaber** [comment]
was this resolved ?
was is the code error ?

File: src/app/not-found.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import Image from 'next/image';
+import Buttons from '@/components/SharedComponents/Buttons';
+import { Lexend_Giga } from 'next/font/google';
+const inter = Lexend_Giga({ subsets: ['latin'] });
```
Comment (jvniorrr) [inline src/app/not-found.tsx]: unneeded import, can be removed

File: src/app/not-found.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+export default function NotFound() {
+	return (
+		<main className="flex min-h-screen flex-col items-center justify-between">
```
Comment (jvniorrr) [inline src/app/not-found.tsx]: for this class name maybe use 
`<main className="flex min-h-screen flex-col items-center justify-between pt-5">`

File: src/app/not-found.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+			<section id="home-section">
```
Comment (jvniorrr) [inline src/app/not-found.tsx]: `<section id="home-section flex flex-col items-center justify-between">`

**Review by jvniorrr — CHANGES_REQUESTED** [review]
Solid work. Working as expected. Just few changes to be incorporated. 

Will also need to rebase your current branch to be synced / aligned with the main branch.

**Comment by jvniorrr** [comment]
Separate PR created, and merged..

**Pull request closed without merging**